lib/vintage_net_wizard/web/router.ex

```diff
@@ -36,7 +36,12 @@ defmodule VintageNetWizard.Web.Router do
   end
 
   get "/ssid/:ssid" do
-    render_page(conn, "configure_password.html", ssid: ssid)
+    password =
+      Backend.configurations()
+      |> Enum.find(%{}, &(&1.ssid == ssid))
+      |> Map.get(:password)
+
+    render_page(conn, "configure_password.html", password: password, ssid: ssid)
```

Comment:
I want to make sure that I understand this. Currently, `vintage_net` doesn't store passwords. It stores the pre-shared key that's derived from the password. It's not possible to reconstruct the password (for mortals, at least). In general, we don't want PSKs or passwords leaving the device to avoid accidentally sharing non-public credentials. Showing what you just typed in is certainly ok - just not what someone else typed earlier for a password field.

Reply:
It looks like we temporarily store the password in memory as part of the config to be applied. I was thinking in case of a password failure, it would be nice to see what you tried before that borked. But I didn't think about if someone else joined the AP and was poking around then could get the password.

Based on what you said, we probably don't want this then. It definitely is what was previously typed in.

Reply:
Removed @fhunleth